exercises/2/webapp/src/components/NewsItem.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <h2>{{ news.title }}({{ news.votes }})</h2>
+    <button @click="upvote">Upvote</button>
+    <button @click="downvote">Downvote</button>
+    <button @click="remove">Remove</button>
+  </div>
+</template>
+<script>
+export default {
+  props: { news: Object },
+  methods: {
+    upvote() {
+      this.$emit("update", { ...this.news, votes: (this.news.votes + 1)});
+    },
+    downvote() {
+      this.news.votes -= 1;
```

Comment:
Why are you updating the news on downvote on a different way compared to upvote?
```suggestion
      this.$emit("update", { ...this.news, votes: (this.news.votes - 1)});
```